packages/tracing-internal/src/node/integrations/express.ts

```diff
@@ -382,11 +478,18 @@ type LayerRoutePathInfo = {
  *          if the route was an array (defaults to 0).
  */
 function getLayerRoutePathInfo(layer: Layer): LayerRoutePathInfo {
-  const lrp = layer.route?.path;
+  let lrp = layer.route?.path;
 
   const isRegex = isRegExp(lrp);
   const isArray = Array.isArray(lrp);
 
+  if (!lrp && !isRegex && !isArray) {
```

Comment:
we don't need to check `isRegex` and `isArray` if `lrp` already coerces to `undefined` right?